packages/app/src/services/extensions.service.ts

```diff
@@ -60,6 +61,16 @@ const extensions: (Extension | DashboardExtension)[] = [
     createdAt: new Date('2024-09-17'),
     isPopular: true,
   },
+  {
+    id: 'HourlyCategoryReport',
+    name: 'Hourly Category Report',
+    authorName: 'Danielle Hovley',
+    authorUrl: 'https://github.com/danzjamz',
```

Comment:
it'd be ideal to include an image as well to help people preview at least the dashboard widget